1001-2000/LIT1957Mashaf.xml

```diff
@@ -33,17 +47,159 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+              ለመምህርየ፡ ዘመጽአ፡ ላዕሌየ፡ ቂመ፡ ወቅንዓተ፡ በእንተ፡ ዘጸልኡኒ፡ ወተጋዓዙኒ፡ አይሁድ፡ በእንተ፡ ማርያም፡ ቅድስት፡ እስመ፡ አልብየ፡
+              አእምሮ፡ በኦሪት፡ ወበነቢያት፡ ወብዙኃን፡ ይጽዕሉኒ፡ በእንተ፡ ፍጥረቱ፡ ለአዳም፡ አቡነ፡ ወፈድፋደ፡ ይጸርፉ፡ ላዕለ፡ እግዚእትነ፡ ማርያም፡
+              ቅድስት፡ ወኢረክቡ፡ ምክንያተ፡ በዘአአትት፡ እከዮሙ፡ ወርኩሰ፡ ቃሎሙ። ወሶበ፡ ነገርክዎ፡ ዘንተ፡ ኵሎ፡ ለመምህርየ፡ ተከዘ፡ ፈድፋደ፡ ወቦአ፡
+              ላዕሌሁ፡ ቅንዓት፡ ወይቤለኒ፡ አነ፡ እከሥት፡ ለከ፡ ኦወልድ፡ ኵሎ፡ ዘሰአልከኒ፡ ወአሌብወከ፡ ዘእምፍጥረተ፡ ዓለም፡ ቀዳሚ፡ ወአጤይቀከ፡ በእንተ፡
+              ቅድስት፡ ማርያም፡ እሙ፡ ለብርሃን፡ ከመ፡ ይእቲ፡ ጥይቅት፡ ዘእንበለ፡ አስትቶ፡ እምትዝምደ፡ ይሁደ፡ ወልደ፡ ያዕቆብ። ወዓዲ፡ እዜንወከ፡ ዘከመ፡
+              ወድቀ፡ ሰይጣን፡ እምሰማይ።
+              </ab>
+            </div>
+          </div>
+          <div type="textpart" subtype="Book" n="1" xml:id="Book1">
+            <label>An account of the creation, and a resumé of Scripture
+              history down to the reign of Jehoram</label>
+
+          </div>
+          <div type="textpart" subtype="Book" n="2" xml:id="Book2">
+            <label>Mysteries revealed by St Peter to Clement</label>
+            <div type="textpart" subtype="incipit" xml:lang="gez">
+              <ab>
+                በስመ፡ አብ፡ <gap reason="ellipsis" resp="PRS10171Wright"></gap> ቀዳሜ፡ ምስጢራት፡
+            ዘከሠተ፡ ቅዱስ፡ ጴጥሮስ፡ ለረድኡ፡ ቀሌምንጦስ። ይቤ፡ ቅዱስ፡ ጴጥሮስ፡ ሶበ፡ ተፈጸመ፡ አርብዓ፡ ዕለት፡
+          እምድኅረ፡ ትንሣኤሁ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ እሙታን፡ ፈቀደ፡ ይዕርግ፡ ውስተ፡ ሰማይ፡
+        ወይትመየጥ፡ ውስተ፡ ስብሐቲሁ፡ ወግሩም፡ ማኅደሩ።
+              </ab>
+            </div>
+          </div>
+          <div type="textpart" subtype="Book" n="3" xml:id="Book3">
+            <label>The power and the wonders which God displayed to St Peter</label>
+            <div type="textpart" subtype="incipit" xml:lang="gez">
+              <ab>
+              በስመ፡ አብ፡ <gap reason="ellipsis" resp="PRS10171Wright"></gap> ንቀድም፡
+              ንጽሐፍ፡ ኀይለ፡ ወተአምረ፡ ዘአርአዮ፡ እግዚአብሔር፡ ለብፁዕ፡ ጴጥሮስ፡ ርእሰ፡ ሐዋርያት፡ ዓዋዴ፡ መንግሥተ፡
+              ሰማያት፡ መጋቢ፡ ዘተወፈየ፡ እምኀበ፡ ሊቁ፡ መራሑተ፡ ጽድቅ፡ ከመ፡ ይፍታሕ፡ ወያርኁ፡ ወያበ፡ ዘፈቀደ፡
+              ለዘኢይደልዎ፡ በጽድቅ፡ ወበርትዕ። ወካዕበ፡ ለዘኢይደልዎ፡ ከመ፡ ይዕፁ፡ ውስተ፡ ገጹ፡ አንቀጸ፡ መንግሥተ፡
+              ሰማያተ፡ ሰማያት።
+              </ab>
+            </div>
+          </div>
+          <div type="textpart" subtype="Book" n="4" xml:id="Book4">
+            <label>Discourse of St Peter to Clement, concerning what shall take place
+            in heaven and on earth for mercy to mankind</label>
+            <div type="textpart" subtype="incipit" xml:lang="gez">
+              <ab>
+                በስመ፡ አብ፡
+              <gap reason="ellipsis" resp="PRS10171Wright"></gap> ንጽሕፍ፡ ዘንተ፡ ነገረ፡ ዘተናገሮ፡
+            ጴጥሮስ፡ ርእሰ፡ ሐዋርያት፡ ለቀሌምንጦስ፡ ተልሚዱ። ዝውእቱ፡ ረድእ፡ ብሂል፡ በትርጓሜሁ፡ ወአለበዎ፡ ዘይከውን፡
+          ዘበሰማይኒ፡ ወዘበምድርኒ፡ በእንተ፡ ምሕረታተ፡ ውሉደ፡ ሰብእ። ወበእንተ፡ አፍቅሮቱ፡ ሰብአ፡ ለመድኃኔ፡ ዓለም፡ እንዘ፡
+        ውስተ፡ ላዕለ፡ ሕፅነ፡ አቡሁ፡ ፍሬ፡ መለኮት፡ ዘመዓዛሁ፡ ሠናይ፡ ወበእንተ፡ ፍቅረ፡ ዚአነ፡ በፈቃደ፡ አቡሁ፡ ወረደ፡
+      ዘበሕፅነ፡ አቡሁ፡ ሠረፀ፡ ፍሬ፡ ስብሐት። ወበመዓዛሁ፡ አምዓዘ፡ ኲሎ፡ ዓለመ።
+              </ab>
+            </div>
+          </div>
+          <div type="textpart" subtype="Book" n="5" xml:id="Book5">
```

Comment:
@DariaElagina I was wondering whether LIT1957Mashaf#Book5 should have a corresp to LIT2680ClemPeter (seems the same text to me)
Also the other parts must be checked if they circulate elsewhere

Reply:
The incipits seems different to me, only one, not yet encoded Synodos ms has some passages from Qalementos

Reply:
OK